### ISSUE TITLE
MLSD [...] command not understood

### ISSUE THREAD

**Issue by brunoqc** [secsy/goftp#2]
I'm trying to use ReadDir() on ftp.openbsd.org and I get :
```
unexpected respone: 500-'MLSD /pub/OpenBSD/snapshots/octeon': command not understood.
```

**Comment by muirrn** [comment]
I wasn't sure how many servers don't support MLSD when I wrote that. I'll add fallback support to use the "LIST" command later tonight.

**Comment by muirmanders** [comment]
Hopefully resolved in 11d2347c5a20ca4c22ce3248e93441a058b968c9

**Comment by brunoqc** [comment]
It works. Thanks!
